credsweeper/utils/util.py

```diff
@@ -95,7 +95,7 @@ def read_file(cls, path: str, encodings: List[str] = default_encodings) -> List[
                     diff_data = file.read().split("\n")
                 break
             except UnicodeError:
-                logging.warning(f"UnicodeError: Can't read content from \"{path}\" as {encoding}.")
+                logging.info(f"UnicodeError: Can't read content from \"{path}\" as {encoding}.")
             except Exception as exc:
                 logging.error(f"Unexpected Error: Can't read \"{path}\" as {encoding}. Error message: {exc}")
```

Comment:
May be we should add warning only if none of the encodings matches.
And, maybe, add info with specific encoding.

TL; DR: I think we should log once per file, not mutliple logs per every encoding